DTS-STN/canadian-dental-care-plan
# Making hCaptcha service injectable

### Description
Also removed references to hcaptcha-service.server and using injectable hCaptcha service.

`h-captcha-route-helper` has been renamed to `hcaptcha-route-helper` for consistency.
This PR can be broken down into smaller PRs upon request.

### Screenshots (if applicable)
![image](https://github.com/user-attachments/assets/64afec0c-1f15-45cf-a481-9d2719088142)

### Checklist

### Test Instructions
Set the logging level to trace. Run application and navigate to `/en/status`. Check for a status.
You should see some logging similar to the screenshot above.
